homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    sql_update_movie_table = """DELETE FROM movie
</pre><p>Do you miss role table?. Delete join table: https://stackoverflow.com/questions/16481379/how-can-i-delete-using-inner-join-with-sql-server</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='148'/>